src/types.ts

```diff
@@ -0,0 +1,50 @@
+export interface EarlResult {
+  '@context': object
+  '@type': 'WebPage'
```

Comment:
I would make `Webpage`, `TestCase` and other custom `string` types, to be an `enum` - https://www.typescriptlang.org/docs/handbook/enums.html

So you could reuse easily like, `MyEnum.WebPage` without worrying about spelling mistakes and wrong typings.